### PR TITLE
Enterprise Release - October 2020

### PR DESCRIPTION
**Pull request by yykamei**

### CODE REVIEW

File: docs/enterprise/releases/202010.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+id: "202009"
+title: Sider Enterprise Release Note in October 2020
+sidebar_label: October 2020
+hide_title: true
```
Comment (yykamei) [inline docs/enterprise/releases/202010.md:5]: > __[remark-lint]__ First heading level should be `1` ([view](https://sider.review/gh/repos/142577083/pulls/516#issue-4875082))
>
> <details>
>
> | Rule | Severity |
> | ---- | -------- |
> | `first-heading-level` | `warn` |
>
> You can [close this issue](https://sider.review/gh/repos/142577083/pulls/516#issue-4875082) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

:man_shrugging:

**Comment by yykamei** [comment]
@ybiquitous @suetsuguk Do you think there are other changes to be included for the new Enterprise release?

File: docs/enterprise/releases/202010.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+id: "202009"
```
Comment (suetsuguk) [inline docs/enterprise/releases/202010.md:2]: The id should be 202010.

Comment (yykamei) [reply docs/enterprise/releases/202010.md]: Thank you. 8c86575

**Comment by suetsuguk** [comment]
I don't have anything to add.

File: docs/enterprise/config.md [hunk 3]
```diff
@@ -129,6 +129,8 @@ The core component **sideci** will invoke Runners to perform analyses. The param
 
   - `no_proxy` - (Optional) The list of domains that should be excluded from the proxy targets. If you set the `http_proxy` and `https_proxy` parameters above, the `no_proxy` parameter may also have to be set. For example, `s3_endpoint` is set with `http://minio:9000`, then `no_proxy` should be set with `minio` if Runners should access to MinIO without a proxy server. See also [HTTP proxy](#http-proxy).
 
+  - `extra_hosts` - (Optional) The list of extra hosts. Use this parameter if you want to add extra lines to `/etc/hosts` in Runners' containers. This parameter is equivalent to the [`--add-host`](https://docs.docker.com/compose/compose-file/#extra_hosts) option of Docker Compose. This parameter should be set as a format of `["hostname:IP"]` if it's specified.
```
Comment (ybiquitous) [inline docs/enterprise/config.md:132]: [question] `--add-host` -> `extra_hosts`?

```suggestion
  - `extra_hosts` - (Optional) The list of extra hosts. Use this parameter if you want to add extra lines to `/etc/hosts` in Runners' containers. This parameter is equivalent to the [`extra_hosts`](https://docs.docker.com/compose/compose-file/#extra_hosts) option of Docker Compose. This parameter should be set as a format of `["hostname:IP"]` if it's specified.
```

File: docs/enterprise/releases/202010.md (hunk 1 above)
Comment (ybiquitous) [reply docs/enterprise/releases/202010.md:5]: #512 may fix the issue?

Comment (yykamei) [reply docs/enterprise/releases/202010.md:5]: Thank you.

**Review by ybiquitous — APPROVED** [review]
Lgtm👌

**Review by suetsuguk — APPROVED** [review]
👍

**Pull request merged**